ext/cosmos/ext/platform/platform.c

```diff
@@ -62,9 +62,8 @@ static VALUE segfault(VALUE self) {
  * Initialize methods for Platform specific C code
  */
 void Init_platform (void) {
-  VALUE ruby_version = rb_const_get(rb_cObject, rb_intern("RUBY_VERSION"));
-
 #ifdef _WIN32
+  VALUE ruby_version = rb_const_get(rb_cObject, rb_intern("RUBY_VERSION"));
```

Comment:
It does work, just causes an unused variable warning.  The ruby_version variable is only used inside of the #ifdef